src/common/base/test/RangeVsTransformBenchmark.cpp

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include <folly/init/Init.h>
+#include <folly/Benchmark.h>
+#include <algorithm>
+#include <vector>
+
+
+BENCHMARK(Test1_RangeTestStr) {
+    std::vector<int32_t> from;
+    std::vector<std::string> to;
+    BENCHMARK_SUSPEND {
+        from.resize(1000, 0);
+        to.resize(1000);
+    }
+    int32_t index = 0;
+    for (auto& i : from) {
+        to[index++] = std::to_string(i);
```

Comment:
You are testing `std::to_string`, not `std::transform` vs. range for.

Reply:
int to std::string is a common case with std::transform.   We'd better to figure out the accurate perf under this case.

Reply:
Fair enough.